firefox-ios/Client/TabManagement/Legacy/LegacyTabManager.swift

```diff
@@ -819,56 +824,96 @@ class LegacyTabManager: NSObject, FeatureFlaggable, TabManager, TabEventHandler
         privateConfiguration.preferences.javaScriptCanOpenWindowsAutomatically = allowPopups
     }
 
-    // returns all activate tabs (private or normal)
-    private func viableTabs(isPrivate: Bool = false) -> [Tab] {
```

Comment:
This method didn't really seem useful anymore. There's a separate task to clean up the inactiveTabs refactor.